specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2020-06-01-preview/apimtenant.json

```diff
@@ -130,15 +183,15 @@
           }
         }
       },
-      "patch": {
+      "put": {
```

Comment:
>put [](start = 7, length = 3)

I assume the remaining changes are just moving the segments around. Please let us know if there are API changes which needs to be reviewed explicitly.

Reply:
Sorry didn't understand the question. In this case we just added the PUT method. Patch method remains unchanged.